main.tf

```diff
@@ -191,6 +191,18 @@ resource "azurerm_application_gateway" "ag" {
     }
   }
 
+  dynamic "ssl_policy" {
```

Comment:
Is this written the simplest way?

Reply:
yupp - I have followed same dynamic blocks logic which are already added on the `azurerm_application_gateway` resource :)